notebooks/20200721_TMPRSS2_SVM.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<p>shouldn't these be .csv, not .zip?  Also, name it something like, 'SVM_results.csv', so it's clearer, and put it in a new folder.  so probably something like <code>active_screening_compounds_sorted.to_csv('../results/SVM_results.csv')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>